playbooks/byo/openshift-glusterfs/uninstall.yml

```diff
@@ -0,0 +1,10 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
```

Comment:
@jarrpa although too late, would have been nice to adhere to ``include_*` or `import_*` to avoid any deprecation messages ...just saying ..

Reply:
Those are enhancements from later versions that are not backported in any other playbook in this directory, so I left it alone. :)